services/common/src/redux/reducers/explosivesPermitReducer.ts

```diff
@@ -14,9 +14,27 @@ const initialState: IExplosivesPermitReducerState = {
 export const explosivesPermitReducer = (state = initialState, action) => {
   switch (action.type) {
     case actionTypes.STORE_EXPLOSIVES_PERMITS:
+      const records = action.payload?.records ?? [];
+      const explosivesPermits = records.map((permit) => {
+        const amendment_count = permit.explosives_permit_amendments.length;
+        const amendments = permit.explosives_permit_amendments.map((a, index) => {
+          const documents = a.documents.map((d) => ({
+            ...d,
+            explosives_permit_document_type_code: d.explosives_permit_amendment_document_type_code,
+          }));
```

Comment:
whyyyy did we give this a different name? It makes bugs. This is one of a few things that maybe should have been a backend fix, but I didn't want to expand scope to include a migration.

Reply:
Yeah, I remember hitting this same point and making the same decision.  I think originally it was due to a lack of a full understanding of what needed to be differentiated here.  The amendments got created as pretty much a straight copy of the permits to start with.